elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemTransformers.java

```diff
@@ -110,6 +110,8 @@ private static void from8(ChainedTransformationDescriptionBuilder chainedBuilder
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.CUSTOM_EVIDENCE_DECODER));
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.AGGREGATE_EVIDENCE_DECODER));
         builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.CERTIFICATE_AUTHORITY));
+        builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.RECONNECT_ATTEMPTS));
+        builder.rejectChildResource(PathElement.pathElement(ElytronDescriptionConstants.SYSLOG_FORMAT));
```

Comment:
Since two new attributes (as opposed to resources) were added, you'll want to use:

```builder.addChildResource(...).getAttributeBuilder().addRejectCheck(...)```